src/mqt/core/evaluation.py

```diff
@@ -0,0 +1,177 @@
+"""Evaluating the json file generated by the benchmarking script."""
+
+from __future__ import annotations
+
+import json
+from pathlib import Path
+from typing import TYPE_CHECKING, Any
+
+import pandas as pd
+
+if TYPE_CHECKING:
+    from os import PathLike
+
+# Avoid output truncation
+pd.set_option("display.max_colwidth", None)
+pd.set_option("display.max_rows", None)
+pd.set_option("display.width", None)
+
+sort_options = ["ratio", "experiment"]
+
+
+def __higher_better(key: str) -> bool:
+    """Return whether a higher value is better for the given key."""
+    higher_better_ls = ["hits", "hit_ratio"]
+    return any(key.endswith(s) for s in higher_better_ls)
+
+
+def __flatten_dict(d: dict[Any, Any], parent_key: str = "", sep: str = "_") -> dict[str, Any]:
+    """Flatten a nested dictionary."""
+    items = {}
+    for key, value in d.items():
+        new_key = f"{parent_key}{sep}{key}" if parent_key else key
+        if isinstance(value, dict):
+            items.update(__flatten_dict(value, new_key, sep=sep))
+        else:
+            items[new_key] = value
+    return items
+
+
+def compare(
+    baseline_filepath: PathLike[Any],
+    feature_filepath: PathLike[Any],
```

Comment:
fairly sure these should be
```suggestion
    baseline_filepath: str | PathLike[str],
    feature_filepath: str | PathLike[str],
```